packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -95,7 +97,7 @@ export async function determineCollectionAndKind(
 
   if (opts?.collection) session.log.debug(`Explicit collection provided: ${opts?.collection}`);
   let selectedCollection = opts?.collection
-    ? openCollections.find((c) => c.name === opts?.collection)
+    ? collections.items.find((c) => c.name === opts?.collection)
```

Comment:
This fixes up the error messaging so user gets `selection is closed` error rather than `selection does not exist`.